students/M_Sunday/lesson04/kata_14.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python
+import os
+import random
+
+
+def openfile(file):
+    text_ = []
+    f = open(file)
+    for word in f.read().split():
+        text_.append(word)
+    for i, word in enumerate(text_):
+        if "." in word:
+            text_[i] = word.replace(".", "")
+        if "," in word:
+            text_[i] = word.replace(",", "")
+        if "(" in word:
+            text_[i] = word.replace("(", "")
+        if ")" in word:
+            text_[i] = word.replace(")", "")
```

Comment:
there is a way to do this in bulk:
```
chars_to_replace = ("(", ")", ",")
for char in chars_to_replace:
    <thetext>.replace(char, "")
```

do the replacement on original text. no reason to loop over lines to create a list and then loop over list again just to replace.